.pylintrc

```diff
@@ -526,10 +526,13 @@ allow-wildcard-with-all=no
 # Analyse import fallback blocks. This can be used to support both Python 2 and
 # 3 compatible code, which means that the block might have code that exists
 # only in one or another interpreter, leading to false positives when analysed.
-analyse-fallback-blocks=no
+analyse-fallback-blocks = yes
```

Comment:
The comment says that it could be used to support py2 & py3 compatible code but do we need it for some other reason?

Reply:
py2/3 is just an example. #8 adds a case with different APIs in different library versions

Reply:
Oh, thx. This comment was unclear for me.